segmentation/include/pcl/segmentation/euclidean_cluster_comparator.h

```diff
@@ -93,14 +95,15 @@ namespace pcl
         * \param depth_dependent
         */
       inline void
-      setDistanceThreshold (float distance_threshold, bool depth_dependent)
+      setDistanceThreshold (const float &distance_threshold,
```

Comment:
Originally I thought: You have only a single copy, inside the body of the function. But then again, when you pass a reference I think you're passing the underlying pointer type into the function. You still have two copies in the end right: 1 pointer + 1 native type. I'll just revert it to what it was.